autogluon/scheduler/remote/remote.py

```diff
@@ -68,7 +68,10 @@ def __init__(self, remote_ip=None, port=None, local=False, ssh_username=None,
         else:
             remote_addr = (remote_ip + ':{}'.format(port))
             self.service = start_service(remote_ip, port)
-            super().__init__(remote_addr)
+            _set_global_remote_service(self.service)
+            import time
+            time.sleep(10)
+            super(Remote, self).__init__(remote_addr)
```

Comment:
Would
super().__init__(remote_addr)
also work here? I recall somebody changed all super calls to this simple format.

And is it OK to call the superclass __init__ in the middle of this method, and in an else branch?

Reply:
Agree. I have changed it to simple format.

I think it is okay to pass different args to __init__ method in an if else branch.